ambry-api/src/main/java/com/github/ambry/network/ChannelOutput.java

```diff
@@ -22,19 +23,23 @@
  */
 public class ChannelOutput {
 
-  private InputStream inputStream;
+  private DataInputStream inputStream;
 
   private long streamSize;
 
-  public ChannelOutput(InputStream inputStream, long streamSize) {
+  public ChannelOutput(DataInputStream inputStream, long streamSize) {
     this.inputStream = inputStream;
     this.streamSize = streamSize;
   }
 
-  public InputStream getInputStream() {
+  public DataInputStream getInputStream() {
     return inputStream;
   }
 
+//  public DataInputStream getDataInputStream() {
+//
+//  }
```

Comment:
remove this?

Reply:
my latest changes were not in. just pushed.